R/transforms.R

```diff
@@ -369,9 +369,9 @@ setMethod("short_description", "TransformRound", function(x) {
 setMethod("round", "IterableMatrix", function(x, digits=0) {
   assert_is(x, "IterableMatrix")
   assert_is(digits, "numeric")
-  if (digits != 0) stop("BPCells only supports round() with digits=0")
+  if (digits < 0) stop("Digits value must be greater than or equal to zero")
```

Comment:
Do we need this restriction? R allows `round(x, -1)` to round to the nearest 10, which I think would work seamlessly with the existing code you've written